api_app/playbooks_manager/migrations/0022_add_dns0_to_free_playbook.py

```diff
@@ -0,0 +1,30 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    playbook_config = apps.get_model("playbooks_manager", "PlaybookConfig")
+    pc = playbook_config.objects.get(name="FREE_TO_USE_ANALYZERS")
+    pc.analyzers.add(*["DNS0_names", "DNS0_rrsets_data", "DNS0_rrsets_name"])
```

Comment:
I'd not put both the `DNS0_rrsets_data` and the `DNS0_rrsets_name` in the same playbook because you are querying the same analyzers. I'd put the second one only.

Reply:
The analyzer is the same, but the query has a different matching type depending on which parameter will be populated by the observable name.

Reply:
yep I got that. The point is that the Free quota is very limited and if you burn 3 api quotas in a single request, it would make these analyzers unusable by an user without a paid api key. Even 2 of them are probably too much.

Reply:
really good point, I'll keep `DNS0_names` and `DNS0_rrsets_name`.